lib/bandit/websocket/connection.ex

```diff
@@ -205,6 +205,10 @@ defmodule Bandit.WebSocket.Connection do
       {:stop, :normal, code, websock_state} ->
         do_stop(code, :normal, socket, %{connection | websock_state: websock_state})
 
+      {:stop, :normal, code, msg, websock_state} ->
+        _ = do_deflate(msg, socket, %{connection | websock_state: websock_state})
```

Comment:
I have two questions about ignoring the return value here:
1. If `do_deflate` succeeds, then we're ignoring updated metrics and `stop_span/2` call within `do_stop/4` will report slightly incorrect results
2. If `do_deflate` fails, we're ignoring a continuation with updated `state: :closed`, so `do_stop/4` will call WebSock `terminate/2` callback a second time on the same connection, and then try to close the socket and span for a second time (not sure if this will cause any errors, haven't dug into `Socket.close/1` code

Mildly incorrect metrics are not the end of the world, but calling `terminate/2` twice may lead to some unexpected behaviour I think

Reply:
Good point about the metrics - I was thinking mostly of failure cases (and, if I'm being honest, just being a bit lazy). 

I'll get those fixed up.

Reply:
Fixed.